### PR TITLE
feat(md): added "import now" to delivery config

### PR DESCRIPTION
**Pull request by ranihorev**
image

### CODE REVIEW

File: packages/core/src/managed/config/DeliveryConfig.tsx [hunk 1]
```diff
@@ -1,24 +1,65 @@
...
+      key: 'import-error',
+      content: `Failed to import delivery config - ${error.message}`,
+      options: { type: 'error' },
+    });
+  }, [error]);
+
+  return (
+    <>
+      <small>
+        (
```
Comment (luispollo) [inline packages/core/src/managed/config/DeliveryConfig.tsx:39]: The parenthesis outside of `<button />` are adding spaces around the text, which looks a bit weird IMHO.

File: packages/core/src/managed/config/DeliveryConfig.tsx [hunk 2]
```diff
@@ -1,24 +1,65 @@
...
+  const appName = useApplicationContextSafe().name;
+  const refetchVariables: FetchApplicationManagementDataQueryVariables = { appName };
+  const [importDeliveryConfig, { error, loading }] = useImportDeliveryConfigMutation({
+    variables: { application: appName },
+    refetchQueries: [{ query: FetchApplicationManagementDataDocument, variables: refetchVariables }],
+  });
+
+  React.useEffect(() => {
+    if (!error) return;
+    NotifierService.publish({
```
Comment (luispollo) [inline packages/core/src/managed/config/DeliveryConfig.tsx:29]: I'm assuming this has an effect on the UI? Can you please post a sample of what that looks like?

File: packages/core/src/managed/graphql/schema.graphql [hunk 3]
```diff
@@ -108,16 +113,16 @@ enum MdLifecycleEventScope {
...
   BUILD
 }
 
 enum MdLifecycleEventStatus {
-  NOT_STARTED,
-  RUNNING,
-  SUCCEEDED,
-  FAILED,
-  ABORTED,
+  NOT_STARTED
```
Comment (luispollo) [inline packages/core/src/managed/graphql/schema.graphql:121]: nit: It's kind of annoying that every time you push changes to the schema it gets reformatted and looks different from what's in Keel...

**Review by luispollo — APPROVED** [review]
Minor style nit and a question, otherwise LGTM.

File: packages/core/src/managed/config/DeliveryConfig.tsx (hunk 2 above)
Comment (ranihorev) [reply packages/core/src/managed/config/DeliveryConfig.tsx:29]: This is the button right notifications you see when an error happens

File: packages/core/src/managed/graphql/schema.graphql (hunk 3 above)
Comment (ranihorev) [reply packages/core/src/managed/graphql/schema.graphql:121]: Fixed it

File: packages/core/src/managed/config/DeliveryConfig.tsx (hunk 2 above)
Comment (ranihorev) [reply packages/core/src/managed/config/DeliveryConfig.tsx:29]: image

**Pull request merged**